packages/expo-splash-screen/android/src/main/java/expo/modules/splashscreen/SplashScreenModule.kt

```diff
@@ -38,6 +48,12 @@ class SplashScreenModule : Module() {
       SplashScreenManager.hide()
     }
 
+    AsyncFunction("_internal_maybeHideAsync") {
```

Comment:
```suggestion
    AsyncFunction("internalMaybeHideAsync") {
```
Same comment for all of these `_internal_` methods